src/rpc/blockchain.cpp

```diff
@@ -1290,8 +1290,8 @@ RPCHelpMan getblockchaininfo()
                                 {
                                     {RPCResult::Type::STR, "status", "one of \"defined\", \"started\", \"locked_in\", \"active\", \"failed\""},
                                     {RPCResult::Type::NUM, "bit", "the bit (0-28) in the block version field used to signal this softfork (only for \"started\" status)"},
-                                    {RPCResult::Type::NUM_TIME, "start_time", "the minimum median time past of a block at which the bit gains its meaning"},
-                                    {RPCResult::Type::NUM_TIME, "timeout", "the median time past of a block at which the deployment is considered failed if not yet locked in"},
+                                    {RPCResult::Type::NUM, "startheight", "the minimum height of a block at which the bit gains its meaning"},
```

Comment:
nit: _"the minimum height of a block at which the bit gains its meaning"_ can be changed to one of the below

_"the minimum height of a block at which the 'bit' gains its meaning"_
_"the minimum height of a block at which the 'bit' becomes relevant"_
_"the minimum height of a block at which the 'bit' used for signaling gains its meaning"_